web/src/app/modules/shared/services/navigation/navigation.service.ts

```diff
@@ -40,6 +57,10 @@ export class NavigationService {
     websocketService.registerHandler('event.octant.dev/navigation', data => {
       const update = data as Navigation;
       this.current.next(update);
+      this.createModules(update.sections);
+      if (update.defaultPath) {
+        this.activeUrl.next(update.defaultPath);
```

Comment:
There's an issue here, where if the defaultPath is missing a leading slash, the url passed to the router will be duplicated. An example case that causes this is switching namespaces via dropdown 2 or more times.

Reply:
The url will now always have leading slash.